### DISCUSSION TITLE
How to downgrade to v3.11.1 ?

### DISCUSSION THREAD

**Discussion by semionline** [hluk/CopyQ#1695] [Q&A]
I'm still at MacOS 10.14.4, and am not in a position to upgrade to 10.15.x.

However, while doing a batch 'brew upgrade' for tons of packages, I also upgraded CopyQ to the latest v4.1.0. This version complains now that it can't run on MacOS 10.14.4 . 

I assumed that, like LInux, upgrading a package always upgrades to the highest version compatible with the version of your OS. Apparently, not so on MacOS.

**Question:** How do I downgrade to v3.11.1 from brew, without having to build from github sources?

Many thanks in advance,

**Comment by hluk** [comment]
Not sure if you can specify a version to install with Homebrew, but you can always get the old version from GitHub releases page: https://github.com/hluk/CopyQ/releases/tag/v3.11.1
